hack/update-vendor-licenses.sh

```diff
@@ -93,15 +93,15 @@ process_content () {
   esac
 
   # Find files - only root and package level
-  local_files=($(
+  IFS=" " read -r -a local_files <<< "$(
```

Comment:
change reason:
```
In ./hack/update-vendor-licenses.sh line 96:
  local_files=($(
               ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```
Ref https://github.com/koalaman/shellcheck/wiki/SC2207

Reply:
Can you please define local files as an array first?

```bash
local_files=()
```